TelCo.ColorCoder/ColorCoderUtil.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using System.Drawing;
+
+namespace TelCo.ColorCoder
+{
+    class ColorCoderUtil
```

Comment:
Be careful with generic names like `...Util`. A reader will not know what exactly it has, so every miscellaneous item will tend to get into it, resulting in a `big ball of mud` after some cycles of updates.
Try naming it specifically based on its functionality = what the users of this class expect from it.

Reply:
I couldn't figure out an appropriate name for the class. Do you have any suggestions?